addons/website/static/src/xml/web_editor.editor.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-inherit="web_editor.LinkDialog" t-inherit-mode="extension" owl="1">
```

Comment:
I guess you wanted to separate the website static templates from the web_editor static templates overrides, but `web_editor.editor.xml` sounds weird. It's `web_editor` already, what is `web_editor.editor` ?